pkg/controller/template/render.go

```diff
@@ -148,6 +148,12 @@ func filterTemplates(toFilter map[string]string, path string, config *RenderConf
 			return nil
 		}
 
+		// Temporarily disable template rendering for all openstack files except "afterburn-hostname.yaml"
+		// This code must be removed when https://github.com/openshift/installer/pull/1959 is merged.
+		if strings.Contains(info.Name(), "openstack") && !strings.Contains(info.Name(), "afterburn-hostname.yaml") {
```

Comment:
The problem is that if it merges, can't have green CI on https://github.com/openshift/installer/pull/1959 as this won't render the templates.

In the patch you reverted, I checked for `config.ControllerConfigSpec.Infra.Status.PlatformStatus.OpenStack` that is set in openshift/installer#1959.

Reply:
Something like `if config.ControllerConfigSpec.Infra.Status.PlatformStatus.OpenStack == nil && strings.Contains(info.Name(), "openstack") && !strings.Contains(info.Name(), "afterburn-hostname.yaml") {` should do it.